src/mito2/src/memtable.rs

```diff
@@ -41,3 +42,39 @@ pub trait MemtableBuilder: Send + Sync + fmt::Debug {
 }
 
 pub type MemtableBuilderRef = Arc<dyn MemtableBuilder>;
+
+// TODO(yingwen): Remove it once we port the memtable.
+/// Empty memtable for test.
+#[derive(Debug, Default)]
+pub(crate) struct EmptyMemtable {
+    /// Id of this memtable.
+    id: MemtableId,
+}
```

Comment:
what's the difference between MetaOnlyMemtable

Reply:
No difference. I move it out of the test_util crate.

Reply:
mv master